css/components/badges.css

```diff
@@ -11,10 +11,10 @@
 .badge__basic-text-xl {
   position: relative;
 }
-.badge__basic-content-s,
-.badge__basic-content-m,
-.badge__basic-content-l,
-.badge__basic-content-xl {
+.badge__basic-content-wrapper,
+.badge__basic-content-wrapper,
+.badge__basic-content-wrapper,
+.badge__basic-content-wrapper {
```

Comment:
If all four are same why do you need four? Must have been a mistake made out of hurry

Reply:
Yeah , thanks for pointing out will fix it soon 👍